package.json

```diff
@@ -37,7 +37,7 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "raw-loader": "^0.5.1",
-    "react": "^0.14.8",
+    "react": ">=0.14.8 <15.2.0",
```

Comment:
so, the reason ^0.14.0 || ^15.0.0 is better is because ^ lets minor and major updates through whereas <15.2.0 would mean that 15.2.1 wouldn't be allowed even though the .1 is only a bugfix patch update